commander/commander.go

```diff
@@ -295,7 +296,16 @@ func bootstrapFromRemoteState(
 		return nil, errors.WithStack(errInconsistentRemoteChainID)
 	}
 
-	return setGenesisStateAndCreateClient(blockchain, storage, fetchedChainState, cfg.Rollup, commanderMetrics)
+	if dbChainState == nil {
+		return setGenesisStateAndCreateClient(blockchain, storage, fetchedChainState, cfg.Rollup, commanderMetrics)
+	}
+
+	if !fetchedChainState.Equal(dbChainState) {
```

Comment:
Do chain states here include "syncedBlock" field? Would it cause problems with the comparison here?

Reply:
There is a field `SyncedBlock`, but it is not compared in the [ChainState.Equal](https://github.com/worldcoin/hubble-commander/blob/239611e4ab05a5cdef22bcdb20063d637003ddc7/models/chain_state.go#L38-L64) method.